.github/workflows/publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish esque
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  publish-to-pypi:
+    name: Publish to pypi
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v1
```

Comment:
I think there is a `v2` version of this action. Not sure what the difference is, though, so you can merge without it :man_shrugging:

Reply:
Good point